php/utils.php

```diff
@@ -1508,19 +1508,25 @@ function get_type( $assoc_args, $arg_types, $default = false ) {
  * # +------+--------+
  * ```
  *
- * @param array $items An array of items to output.
+ * @param array $items   An array of items to output.
+ * @param bool $log_data To log table in file or not.
  *
  */
-function format_table( $items ) {
+function format_table( $items, $log_in_file = false ) {
 	$item_table = new \cli\Table();
 	$item_table->setRows( $items );
 	$item_table->setRenderer( new \cli\table\Ascii() );
 	$lines = array_slice( $item_table->getDisplayLines(), 3 );
 	array_pop( $lines );
 	$delem = $item_table->getDisplayLines()[0];
 	foreach ( $lines as $line ) {
-		\EE::log( $delem );
-		\EE::log( $line );
+		if ( $log_in_file ) {
```

Comment:
Why do you keep checking this in the loop? Keep it simple and move it outside loop.
```php
if ( $log_in_file ) {
	foreach ( $lines as $line ) {
		\EE::log( $delem );
		\EE::log( $line );
	}
} else {
	foreach ( $lines as $line ) {
		\EE::line( $delem );
		\EE::line( $line );
	}
}
```

Reply:
Handled in https://github.com/EasyEngine/easyengine/pull/1211